numpy/f2py/_backends/src/meson.build.src

```diff
@@ -0,0 +1,40 @@
+project('${modulename}',
+        ['c', 'fortran'],
+        version : '0.1',
+        meson_version: '>= 1.1.0',
+        default_options : ['warning_level=2'])
+
+py_mod = import('python')
+py = py_mod.find_installation(pure: false)
```

Comment:
Minor: newer practice is to put this on a single line because `py_mod` isn't needed for anything:
```meson
py = import('python').find_installation(pure: false)
```